README.md

```diff
@@ -1,28 +1,33 @@
 # Dss Flappers
 
-Implementations of MakerDao surplus auctions, triggered on `vow.flap`. The current featured Flapper is `FlapperUniV2`.
+Implementations of MakerDao surplus auctions, triggered on `vow.flap`.
```

Comment:
Need to update the documentation about `FlapperMom` (replace with `SplitterMom` etc..).